models/staging/education/staging_teacher_satisfaction_survey.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+  materialized='table'
+) }}
+
+WITH source_data AS (
+    SELECT
+        id,
+        data::jsonb AS json_data  -- Cast the data column to JSONB
```

Comment:
Same question, do we really need this if the data column is already in jsonb format. 

I might be wrong too.